Resources/public/js/views/tabs/ez-locationviewversionstabview.js

```diff
@@ -112,12 +115,12 @@ YUI.add('ez-locationviewversionstabview', function (Y) {
         },
 
         /**
-         * Enables the `buttons corresponding to the archived list.
+         * Enables the Archived buttons if the selection in the list is right
          *
-         * @method _enableArchivedVersionButtons
+         * @method _setArchivedButtonsState
          * @protected
          */
-        _enableArchivedVersionButtons: function () {
+        _setArchivedButtonsState: function () {
```

Comment:
just make sure `_enableArchivedVersionButtons` was not in the last release, otherwise you'll have to keep it as a deprecated method.